### PR TITLE
Revert unnecessary changes vs upstream in fd.c

### PR DESCRIPTION
**Pull request by hlinnaka**
To make merging easier.

@gcaragea, would you happen to remember the reason for the error message changes by any chance? I looked at the old git history before open sourcing, and the "file descriptor" -> "file handle" changes were made back in 2011 by your commit 670978c3, but I didn't see any rationale for the change.

### CODE REVIEW

**Comment by gcaragea** [comment]
Heikki, that was part of a larger error message clean-up, but I don't remember the rationale for this particular change. Reverting back to upstream makes sense, I hope it doesn't break any tests that diff on the error message though..

**Comment by gcaragea** [comment]
LGTM

File: src/backend/storage/file/fd.c [hunk 1]
```diff
@@ -969,20 +976,23 @@ OpenNamedFile(const char   *fileName,
...
 
 	/*
-	 * File flags when open the file.  Note: we don't use O_EXCL, in case there is an orphaned
+	 * Open the file.  Note: we don't use O_EXCL, in case there is an orphaned
 	 * temp file that can be reused.
 	 */
-	int fileFlags = O_RDWR | PG_BINARY;
+	fileFlags = O_RDWR | PG_BINARY;
 	if (create)
-	{
```
Comment (asimrp) [inline src/backend/storage/file/fd.c]: Surprised to see upstream getting rid of the enclosing braces.  Without them, it's easy to fall into misleading indentation trap.  I recently stumbled upon `-wmisleading-indentation` feature addition to GCC 6: http://developerblog.redhat.com/2016/02/26/gcc-6-wmisleading-indentation-vs-goto-fail/

**Comment by asimrp** [comment]
LGTM

**Pull request merged**